packages/icons-angular/src/story-templates.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright IBM Corp. 2018, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const { param, pascal } = require('change-case');
+
+const iconStoryTemplate = icon => `.add("${icon.moduleName}", () => ({
+  template: \`
+    <p>Component <code>&lt;ibm-icon-${param(
+      icon.moduleName
+    )}&gt;&lt;/ibm-icon-${param(icon.moduleName)}&gt;</code></p>
+    <ibm-icon-${param(icon.moduleName)}></ibm-icon-${param(icon.moduleName)}>
+    <p>Directive <code>&lt;svg ibmIcon${pascal(
+      icon.moduleName
+    )}&gt;&lt;/svg&gt;</code></p>
+    <svg ibmIcon${pascal(icon.moduleName)}></svg>
+  \`
+}))
+.add("${icon.moduleName} with label", () => ({
+  template: \`
+    <ibm-icon-${param(
+      icon.moduleName
+    )} ariaLabel="label for the icon"></ibm-icon-${param(icon.moduleName)}>
+    <svg ibmIcon${pascal(icon.moduleName)} ariaLabel="label for the icon"></svg>
+  \`
+}))
+.add("${icon.moduleName} with title", () => ({
+  template: \`
+    <ibm-icon-${param(icon.moduleName)} title="icon title"></ibm-icon-${param(
+  icon.moduleName
+)}>
+    <svg ibmIcon${pascal(icon.moduleName)} title="icon title"></svg>
+  \`
+}))
+.add("${icon.moduleName} with class on the SVG", () => ({
+  template: \`
+    <ibm-icon-${param(
+      icon.moduleName
+    )} innerClass="test-class another-class"></ibm-icon-${param(
+  icon.moduleName
+)}>
+    <svg ibmIcon${pascal(
+      icon.moduleName
+    )} class="test-class another-class"></svg>
+  \`
+}))`;
+
+const gerateIconStories = icons => {
```

Comment:
is this intentional?

Reply:
It was ... though we could probably replace it with a `reduce` 🤔

Reply:
oh I was referring more to the name but a `reduce` would work here as well yeah, either approach is fine

Reply:
Oh wow, I totally didn't see that 😅 yeah that should definitely be _generate_ - but I'll probably swap it out for a `reduce` anyway

Reply:
done!